R/component.R

```diff
@@ -23,7 +23,9 @@
 #'      (excluding aliased coefficients).
 #' - `beta_vcov_complete`: estimated variance-covariance matrix including
 #'      aliased coefficients with entries set to `NA`.
-#' - `varcor`: estimated covariance matrix for residuals.
+#' - `varcor`: estimated covariance matrix for residuals. If there are multiple
+#'      groups, a named list of estimated covariance matrix for residuals will be
```

Comment:
Just adding plural here. 

```suggestion
#'      groups, a named list of estimated covariance matrices for residuals will be
```